src/Entity/SmsMessage.php

```diff
@@ -136,6 +144,21 @@ public function removeOption($name) {
   /**
    * {@inheritdoc}
    */
+  public function getResult() {
```

Comment:
Yeh I though about this too, but I'd probably wait until if more methods justify a trait. Having one method in a trait would only serve for confusion in the future since I'd likely forget about it. I doubt we would be saving much LOC, due to the boilerplate.

So, maybe another time?